fog/report/api/src/lib.rs

```diff
@@ -14,17 +14,50 @@ mod autogenerated_code {
     include!(concat!(env!("OUT_DIR"), "/protos-auto-gen/mod.rs"));
 }
 
+use crate::report::Report_oneof_attestation_evidence;
 pub use autogenerated_code::*;
+use mc_attest_verifier_types::{prost, VerificationReport};
+use mc_fog_report_types::AttestationEvidence;
 
 // These are needed for tests
 impl Eq for report::Report {}
 impl Eq for report::ReportResponse {}
 
+impl From<Report_oneof_attestation_evidence> for AttestationEvidence {
+    fn from(value: Report_oneof_attestation_evidence) -> Self {
+        match value {
+            Report_oneof_attestation_evidence::dcap_evidence(evidence) => {
+                prost::DcapEvidence::try_from(&evidence)
```

Comment:
A bit out of the ordinary to swallow the error but I think the headache of changing everything to `try_from` might make this an acceptable situation (and we can potentially address in a followup).

Reply:
This has been redone to use a `from()` via https://github.com/mobilecoinfoundation/mobilecoin/pull/3592